spec/models/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+RSpec.describe User, type: :model do
+  it "is expected to have valid Factory" do
+    expect(create(:user)).to be_valid
```

Comment:
You also have two other factories. Are you resting these?

Reply:
We added tests for :registered_user and :journalist too